qt/python/mantidqt/widgets/sliceviewer/view.py

```diff
@@ -44,33 +47,39 @@ def __init__(self, presenter, dims_info, can_normalise, parent=None):
         self.line_plots = False
         self.can_normalise = can_normalise
         self.nonortho_tr = None
+        self.ws_type = dims_info[0]['type']
 
         # Dimension widget
+        self.dimensions_layout = QGridLayout()
         self.dimensions = DimensionWidget(dims_info, parent=self)
         self.dimensions.dimensionsChanged.connect(self.presenter.dimensions_changed)
         self.dimensions.valueChanged.connect(self.presenter.slicepoint_changed)
+        self.dimensions_layout.addWidget(self.dimensions, 0, 0, 2, 1)
 
         self.colorbar_layout = QVBoxLayout()
-        self.colorbar_layout.setContentsMargins(0,0,0,0)
+        self.colorbar_layout.setContentsMargins(0, 0, 0, 0)
         self.colorbar_layout.setSpacing(0)
 
+        self.image_info_widget = ImageInfoWidget(self.ws_type, self)
+        self.track_cursor = QCheckBox("Track Cursor", self)
```

Comment:
Could we add a tooltip describing what this does? If you've not used it before it's not immediately obvious that clicking when this is unchecked updates the table.